docs/detections/osquery-response-action.asciidoc

```diff
@@ -0,0 +1,62 @@
+[[osquery-response-action]]
+=== Add Osquery Response Actions
+Osquery Response Actions allow you to add live queries to custom query rules so you can automatically collect data on systems the rule is monitoring. Use this data to support your alert triage and investigation efforts.
+
+NOTE: Response Actions are in technical preview.
+
+[role="screenshot"]
+image::images/available-response-actions.png[Shows available response actions]
```

Comment:
"Shows" is unnecessary for alt text; every image shows something, so we don't need to say it every time. Most alt text guidelines I've seen (Elastic's [accessibility guidelines](https://docs.elastic.dev/content-design/accessibility#guidelines) and other [industry guidelines](https://webaim.org/techniques/alttext/)) just recommend simple descriptions, no verbs.